cql3/statements/create_table_statement.cc

```diff
@@ -135,7 +135,7 @@ schema_ptr create_table_statement::get_cf_meta_data(const database& db) const {
 void create_table_statement::apply_properties_to(schema_builder& builder, const database& db) const {
     auto&& columns = get_columns();
     for (auto&& column : columns) {
-        builder.with_column(column);
+        builder.with_column_ordered(column);
```

Comment:
Note that it's still kind of inaccurate since we'll ignore the order for regular/static columns. But it's better than before, and more precision won't help much here.